tests/integration-test/bpf/build_push_images.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
Good question and I agree that documentation is needed.  I was debating with myself whether to add it.  To make these IT updates, I had to figure out how to build and push the images, and also figure out the relative paths to build things right.  I think this will need to be done in the future, so it seemed useful to preserve it.

We eventually need to push the images automatically (issue #533) when code gets merged.  However, it's still useful to be able to push them manually.  For example, if you want to modify the eBPF code in an existing test program and then update the integration tests to use the modified code, this makes it easier to build and push them.  Of course, anyone doing this should make sure that any changes are backward compatible with the existing integration tests.  Also, when we add new programs or program types, it's a good template to use and to push them so that they can be used in new tests.  

I guess this could also be done with `cargo xtask`, but I'd have to figure out how to do that.  So, if we do that in the future, this script could be removed.

How about a README file in this directory?

Reply:
Maybe add a new file `docs/developer-guide/testing.md`. We can add other sections later, but start with just an intergration test section. Since all our other document is under `doc/`, I'm afraid it would get lost as a README under a lower directory.